README.md

```diff
@@ -69,6 +69,20 @@ To build binary: ```mvn package```
 - which will produce  `tool/target/datacommons-import-tool-0.1-alpha.1-jar-with-dependencies.jar`
 - and you can run it with ```java -jar <path-to-datacommons-import-tool-jar-with-dependencies.jar>```
 
+### Run Server
```

Comment:
Maybe indicate that this is an experimental server for private DC?

Reply:
Done